vpr/src/device/rr_graph_view.h

```diff
@@ -96,6 +96,72 @@ class RRGraphView {
         return node_storage_.fan_in(node);
     }
 
+    /* Get the xlow of a routing resource node. This function is inlined for runtime optimization. */
```

Comment:
1. It would be good to briefly explain what xlow is (minimum x-coordinate) is here.  Same for the other four coordinates.
2. Make these comments Doxygen-compatible, so we can document this API in the VTR readthedocs documentation.  That could be done in a separate PR though.